src/options.html

```diff
@@ -20,12 +20,5 @@
 		</label>
 	</p>
 </form>
-<form id="custom-domain">
-	<h2>GitHub Enterprise support</h2>
-	<p class="js-permission-api">
-		<input type="url" placeholder="https://github.yourdomain.com" size="30" required id="custom-domain-origin">
-		<button type="submit">Authorize</button>
-	</p>
-</form>
```

Comment:
Can you preserve the heading and just replace input with a link to instructions? When you do the reusable module, the link could just link to the instructions there.

Reply:
Excellent! Will do

Reply:
Options can include more info, generally, like a link to `readme.md#customization`, a link to `label:help wanted` and whatever else could be useful.

Reply:
<img width="451" alt="GHE info in Options" src="https://user-images.githubusercontent.com/1402241/32874912-16f3522e-cad0-11e7-9f31-3927b28aeed4.png">